metaflow/plugins/kfp/s3_sensor_decorator.py

```diff
@@ -68,24 +68,29 @@ class S3SensorDecorator(FlowDecorator):
         "timeout_seconds": 3600,
         "polling_interval_seconds": 300,
         "path_formatter": identity_formatter,
+        "os_vars": False,
```

Comment:
Can you please provide an example in this file of using `os_expandvars`, because the syntax is different